test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/V3DetailControlModelTests.cs

```diff
@@ -50,6 +52,28 @@ public V3DetailControlModelTestBase(GlobalServiceProvider sp, V3PackageSearchMet
                 Sources = new List<PackageSourceContextInfo> { new PackageSourceContextInfo("nuget.psm.test") },
             };
         }
+
+        /// <summary>
+        /// Due to embedding the types we need to compare based on IsEquivalentTo
```

Comment:
Embedding as in "embed interop types"?  What types?

Reply:
Zdrasvuyte Maya Druga

Reply:
I didn't write this comment. I'm also unclear what is meant by "embedding". Also potentially this is dead code? I don't see any calls that use the comparer.